src/doc/java_client/index.rst

```diff
@@ -38,45 +37,268 @@ What follows are examples of how VisIt_'s Java client can be used.
+``-stay``: 
+    tells the program to continue running after the ``work()`` method has completed instead of exiting.
 
-`-path`: 
+``-path``: 
     path to VisIt_'s bin directory, eg, ``/usr/local/visit/3.3.0/bin``
 
-`-datapath`: 
+``-datapath``: 
     path to directory holding VisIt_'s silo example data (used by many of the examples)
     eg: ``/usr/local/visit/3.3.0/data/``. **Note**: the trailing slash is **required**.
 
-`-dv`: 
+``-dv``: 
     Tells the example application that it will be connecting to a development version of VisIt_.
     This is important because an installed version of VisIt_ is structured slightly differently than a build.
 
-All of the examples are available from the the top (``.``) directory of the untarred ``jvisit<version>.tar.gz``.
+``-nopty``:
+    Is forwarded to VisIt_ launcher to make VisIt_ prompt for passwords in the console/terminal from which Java was run.
+
+All of the examples are available from the top (``.``) directory of the untarred ``jvisit<version>.tar.gz``.
 
 Java Classes Documentation
 --------------------------
 
 There is a ``docs`` subdirectory from the untarred ``jvisit<version>.tar.gz`` with a bunch of ``.html`` files.
 If you point a web browser at ``index.html`` there, you can find a lot of documentation on the various Java classes available and used in various of these examples.
 
-.. toctree::
-   :maxdepth: 1
-
-   Basic example                              <Examples/RunViewer>
-   Controlling lighting                       <Examples/TryLighting>
-   Performing queries                         <Examples/TryQuery>
-   Getting metadata                           <Examples/GetMetaData>
-   Controlling annotations                    <Examples/TryAnnotations>
-   Making host profiles                       <Examples/MakeHostProfile>
-   Opening the visIt GUI from Java            <Examples/OpenGUI>
-   Determining which variables can be plotted <Examples/PlotTypes>
-   Executing Python from Java                 <Examples/DualClients>
-   Plotting vectors from Java                 <Examples/PlotVector>
-   Changing plot attributes                   <Examples/PlotAtts>
-   Changing points size and shape             <Examples/TryPointGlyphing>
-   Using Threshold operator                   <Examples/TryThreshold>
+Running the Examples
+--------------------
+
+Assuming the current working directory is the directory where ``jvisit<version>.tar.gz`` was untarred, to run an example named ``Example.class``, do the following...
+
+.. code-block:: shell
+
+    java -cp .:visit.jar Example -stay -path /path/to/visit/bin -datapath /path/to/silo/data/dir/
+
+.. note:: The Java program to run, here ``Example``, does not include the ``.class`` extension when it is specified on the command line to ``java``.
+   The trailing slash for the ``-datapath`` argument is required.
+   From within a development build of VisIt_, the ``-path`` option and its argument may be replaced with ``-dv``.
```

Comment:
"-dv" is not the same as "-path". "-dv" means you are working from a development version of VisIt, and currently the examples then expect that "../bin" respective to the location of the java example is where "visit" executable is located.  That is certainly not the case on Windows, and may not be the case on Linux.   I think that it is better to say that "-dv" without "--path" assumes "../bin" is where the visit executable is located, and that "--path" should be used if that is not the case.

Reply:
Ok, I did basically that but I did it up where `-dv` was first documented in the field list.